README.md

```diff
@@ -100,57 +99,43 @@ make
 
 ## For Windows:
 
-1. Please download and install OpenSSL for Windows.
-
-The 64-bit devel package can be downloaded from here:
-
-     http://slproweb.com/download/Win64OpenSSL-1_0_2r.exe
+### 1. Download and install OpenSSL for Windows.
 
+The 64-bit package can be downloaded from here: [Win64OpenSSL_Light-1_1_1c.exe](http://slproweb.com/download/Win64OpenSSL_Light-1_1_1c.exe).
 	 
 (Note that the last letter or version number may be changed and older versions
 no longer available. If this isn't found, check here:
 http://slproweb.com/products/Win32OpenSSL.html
 )
 
-It's expected to be installed in `C:\OpenSSL-Win64` (see the above variables).
-
-Note that this version is compiled most likely for Visual Studio 2013. For
-other versions you better download and compile the sources by yourself,
-from: https://github.com/openssl/openssl
-
-The instruction for Windows:
-http://developer.covenanteyes.com/building-openssl-for-visual-studio/
-
-2. Compile and install Pthreads for Windows from this submodule:
+It's expected to be installed in `C:\Program Files\OpenSSL-Win64`.
+Add this path to the user's or system's environment variable `PATH`.
 
-     submodules/pthread-win32
-	 
-Please follow the steps:
 
-a. Using Visual Studio 2013, please open this file:
+### 2. Install Pthreads for Windows
 
-     pthread_lib.2013.vcxproj
+nuget install cinegy.pthreads-win64 -version 2.9.1.17 -OutputDirectory C:\pthread-win32
```

Comment:
More about how to have `nuget` available would be desired.

Reply:
I agree. How about something simple:
"Instructions on how to install NuGet package can be found at the following location: https://docs.microsoft.com/en-ca/nuget/install-nuget-client-tools"

Reply:
Why require a package manager just for this? You can clone and build https://github.com/GerHobbelt/pthread-win32 trivially